### PR TITLE
silitski_hometask/5

### PR DESCRIPTION
**Pull request by qqqwww123cdv**
Finished 5 homework.

### CODE REVIEW

File: qqqwww123cdv/hometask-5/3.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SELECT last_name || '-' || first_name AS full_name
+FROM actor
+WHERE first_name LIKE 'W%' OR last_name LIKE '%H';
```
Comment (jekapulpit) [inline qqqwww123cdv/hometask-5/3.sql]: I don't think that the anyone's last name ends on `H`, it's `h` probably 😄

File: qqqwww123cdv/hometask-5/4.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SELECT DISTINCT rental_duration, COUNT(title) AS in_each_group
+FROM film
+GROUP BY 1;
```
Comment (jekapulpit) [inline qqqwww123cdv/hometask-5/4.sql]: let's be more clear in the grouping section here, just a number brings some confusion

File: qqqwww123cdv/hometask-5/4.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+SELECT DISTINCT rental_duration, COUNT(title) AS in_each_group
```
Comment (jekapulpit) [inline qqqwww123cdv/hometask-5/4.sql]: why do we need `DISTINCT` here?

File: qqqwww123cdv/hometask-5/5.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+SELECT name, COUNT(film_id) AS number_of_films
```
Comment (jekapulpit) [inline qqqwww123cdv/hometask-5/5.sql]: let's specify column names in front of fields if we have more than one table in our selection

**Review by jekapulpit — CHANGES_REQUESTED** [review]

**Review by jekapulpit — DISMISSED** [review]

**Review by jekapulpit — APPROVED** [review]

**Comment by qqqwww123cdv** [comment]
silitski_hometask/5

**Pull request merged**